runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpInfoBox.java

```diff
@@ -96,6 +97,8 @@
 
 	private final XpTrackerConfig xpTrackerConfig;
 
+	@Getter(AccessLevel.PACKAGE)
+	@Setter(AccessLevel.PACKAGE)
```

Comment:
This do not needs to be exposed anymore right, now that you are using XpPauseState